.github/workflows/config.yml

```diff
@@ -20,7 +20,7 @@ env:
   OPENSTUDIO_PLATFORM: Ubuntu-22.04-x86_64
   OPENSTUDIO_EXT: deb
   OPENSTUDIO_URL: https://github.com/NREL/OpenStudio/releases/download/v3.9.0
-  BUILDSTOCK_BRANCH: develop
+  BUILDSTOCK_BRANCH: pp_fix
```

Comment:
Does this need to get changed back before merging?

Reply:
Yes, will switch these back before merging.